### PR TITLE
ER-284 Wage section

### PR DESCRIPTION
**Pull request by chrisjensenuk**
- Ensuring dates must be yyyy
- Added Part1 section. Wages
- Updated validation messages as specified
- Using `<input type="number" />` 
- Ensuring custom validators only run if a value has been specified

### CODE REVIEW

File: src/Employer/Employer.Web/Views/Part1/Wage/Wage.cshtml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        <form asp-route="RouteNames.Wages_Get">
+
+            <div class="form-group">
+                <fieldset>
+                    <label asp-for="Duration" class="form-label-bold">How long is the apprenticeship expected to last?</label>
+                    <span asp-validation-for="Duration" class="error-message"></span>
+                    <input asp-for="Duration" class="form-control form-control-1-8" type="number">
+                    <select asp-for="DurationUnit" class="form-control" style="width: auto">
+                        <option value="@DurationUnit.Month">@DurationUnit.Month.ToString()</option>
+                        <option value="@DurationUnit.Year">@DurationUnit.Year.ToString()</option>
```
Comment (sachpatel) [inline src/Employer/Employer.Web/Views/Part1/Wage/Wage.cshtml]: Could we display `Month(s)` and `Year(s)` here.

File: src/Employer/Employer.Web/Views/Part1/Wage/Wage.cshtml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            </h2>
+        </div>
+        
+        <form asp-route="RouteNames.Wages_Get">
+
+            <div class="form-group">
+                <fieldset>
+                    <label asp-for="Duration" class="form-label-bold">How long is the apprenticeship expected to last?</label>
+                    <span asp-validation-for="Duration" class="error-message"></span>
+                    <input asp-for="Duration" class="form-control form-control-1-8" type="number">
```
Comment (sachpatel) [inline src/Employer/Employer.Web/Views/Part1/Wage/Wage.cshtml]: I am able to put negative numbers here and submit the form is other values are valid. It is not in AC criteria.

Comment (chrisjensenuk) [reply src/Employer/Employer.Web/Views/Part1/Wage/Wage.cshtml]: This falls under a business rule so will be addressed in a later ticket

File: src/Employer/Employer.Web/Views/Part1/Wage/Wage.cshtml (hunk 1 above)
Comment (chrisjensenuk) [reply src/Employer/Employer.Web/Views/Part1/Wage/Wage.cshtml]: 👍

**Pull request merged**